src/modules/layout/components/Toolbar.js

```diff
@@ -24,22 +28,38 @@ const ToolbarContainer = styled.div`
 
   @media ${props => props.theme.dimensions.smallViewport} {
     width: 100%;
-    flex-direction: row;
+    flex-flow: row wrap;
     justify-content: center;
   }
 `
 
+const Headline = styled.h5`
+  width:100vw;
+  margin: 0;
+  text-align: center;
+  font-size: 90%;
+`
+
 type PropsType = {|
   className?: string,
-  children?: Node
+  children?: Node,
+  viewportSmall: boolean,
+  t: TFunction
 |}
 
+const mapStateToProps = (state: StateType) => ({
+  viewportSmall: state.viewport.is.small
+})
+
 class Toolbar extends React.PureComponent<PropsType> {
   render () {
+    const { viewportSmall, t } = this.props
+    console.log('switcher', viewportSmall)
     return <ToolbarContainer className={this.props.className}>
+      {viewportSmall && <Headline>{t('isThisSiteUseful')}</Headline>}
       {this.props.children}
     </ToolbarContainer>
   }
 }
 
-export default Toolbar
+export default withTranslation('feedback')(connect<*, *, *, *, *, *>(mapStateToProps)(Toolbar))
```

Comment:
Avoid connecting a component to the state. Instead only containers should be connected to the state.

Reply:
This has been reverted by using props propagation from parent components now.